### PR TITLE
Add two examples for assessment and comparison

### PR DESCRIPTION
**Pull request by xyxxxxx**
两个示例，一个是直接运行 Workflow（第一步下载和预处理数据集，第二步创建 Job），另一个是提供详细的操作步骤，用 kubectl 创建 Pod 和 Job

先看看哪种形式好 @ppkube

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
+
+TensorStack AI 计算平台所在的 Kubernetes 集群需要：
+
+* 拥有足够的 AI 计算资源：拥有至少一个这样的节点，其拥有至少 4 个 A100 或同等级别的 GPU，一定量的 CPU（核心数）和内存（不同的 benchmark 使用的 CPU 和内存资源量不同，请参照各个 benchmark 的 README）。
+* 拥有足够的存储资源：数个 TiB 的存储容量。
+* 连接到互联网。
+
+## 使用方法
+
+1. 创建一个名为 mlperf 的队列，其扩展资源 `nvidia.com/gpu` 的值填写您要使用的所有节点的 GPU 数量之和（至少为 4），`CPU 额度` 和 `内存额度` 分别填写这些节点的 CPU 和内存量之和，`节点筛选` 添加适当的标签匹配规则（例如匹配 `sched.tensorstack.dev/accelerator: nvidia-gpu`、`sched.tensorstack.dev/node-alloc-mode: nvidia`、`nvidia.com/gpu.product: NVIDIA-A100-SXM-80GB`、`nvidia.com/gpu.memory: 81920` 等）来限制队列只能使用这些节点。
```
Comment (ppkube) [inline README.md]: 1. queue is optional
2. if queue is used, it's up to the user to choose a suitable queue with **sufficient** resources and **proper** settings. 

So, we only need to document what resources/settings are needed here, not how to set up a queue.

File: image-segmentation/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# cd into current directory
+cd ~/mlperf-examples/image-segmentation
+kubectl apply -f workflow_DGXA100_40GB_4.yaml
+```
+
+文件名中的 `DGXA100_40GB` 表示配置适用于 NVIDIA A100 GPU（40GB 显存），`4` 表示 GPU 数量。如果您的硬件有所不同，请参阅[配置和指标](#配置和指标)部分以修改配置。
+
+创建 WorkflowRun 之后，前往工作流控制台查看其运行进度。第一次运行时，拉取镜像可能花费较长时间。
+
+## 细节
```
Comment (ppkube) [inline image-segmentation/README.md:25]: Please briefly describe the purpose of each `WorkflowTemplate` and `WorkflowRun`

File: README.md [hunk 3]
```diff
@@ -1 +1,28 @@
...
+MLPerf™ Training benchmark 是一个机器学习领域的基准测试套件，旨在针对现代机器学习工作负载公正、准确、全面地评估系统性能。更多详细信息请参阅官方网站 [MLCommons v2.1 Results](https://mlcommons.org/en/training-normal-21/)以及论文 [MLPerf Training Benchmark](https://mlcommons.org/en/training-normal-21/)。
+
+本项目提供 MLPerf Training 最新版本（当前为 v2.1）的各个 benchmark 在 TensorStack AI 计算平台上的运行方法。所有模型的实现均采用 [NVIDIA 的提交](https://github.com/mlcommons/training_results_v2.1/tree/main/NVIDIA)。
+
+## 使用前提
+
+TensorStack AI 计算平台所在的 Kubernetes 集群需要：
+
+* 拥有足够的 AI 计算资源：拥有至少一个这样的节点，其拥有至少 4 个 A100 或同等级别的 GPU，一定量的 CPU（核心数）和内存（不同的 benchmark 使用的 CPU 和内存资源量不同，请参照各个 benchmark 的 README）。
+* 拥有足够的存储资源：数个 TiB 的存储容量。
```
Comment (ppkube) [inline README.md:12]: Document that a PVC named `mlperf` with sufficient size needs to be created first.

File: image-segmentation/workflow_DGXA100_40GB_4.yaml [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+                    - mountPath: /pvc
+                      name: data
+                    - mountPath: /dev/shm
+                      name: dshm
+                    - mountPath: /results
+                      name: results
+              volumes:
+                - name: data
+                  persistentVolumeClaim:
+                    claimName: mlperf
```
Comment (ppkube) [inline image-segmentation/workflow_DGXA100_40GB_4.yaml]: PVC name `mlperf` is hard-coded in this MPIJob.  This poses a problem as  changing the value of the parameter `pvc_name` in workflowrun `mlperf-segmentation` won't work.

File: reinforcement-learning/mpijob.yaml [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+              limits:
+                cpu: 16
+                memory: 96Gi
+                tensorstack.dev/nvidia-gpu: 1
+            volumeMounts:
+              - mountPath: /pvc
+                name: data
+              - mountPath: /dev/shm
+                name: dshm
+              - mountPath: /results
```
Comment (ppkube) [inline reinforcement-learning/mpijob.yaml:76]: what's this `results` for? why using emptyDir?

File: reinforcement-learning/mpijob.yaml [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+  name: minigo
+spec:
+  mca:
+    btl: ^openib
+    pml: ob1
+  mpiHome: /usr/local/mpi
+  ssh:
+    sshdPath: /usr/sbin/sshd
+  scheduler:
+    t9kScheduler:
```
Comment (ppkube) [inline reinforcement-learning/mpijob.yaml]: 1. keep `scheduler` in comments. 
2. use a consistent queue name, e.g. `default`?

File: image-segmentation/workflow_DGXA100_40GB_4.yaml [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+      metadata:
+        name: mlperf-segmentation
+      spec:
+        mca:
+          btl: ^openib
+          pml: ob1
+        mpiHome: /usr/local/mpi
+        ssh:
+          sshdPath: /usr/sbin/sshd
+        scheduler:
```
Comment (ppkube) [inline image-segmentation/workflow_DGXA100_40GB_4.yaml]: Don't use  `scheduler` unless a user chooses so. For convenience, we can keep the scheduler settings as  inline comments.

**Comment by ppkube** [comment]
> 先看看哪种形式好 @ppkube

1. as the first iteration,  let's  "提供详细的操作步骤，用 kubectl 创建 Pod 和 Job" for all new mlperf examples, as it takes less short-term effort. 
2. For the example `image_segementation` that's already supported by workflow, keep it as is.

**Review by ppkube — CHANGES_REQUESTED** [review]
various minor issues need to be fixed

File: README.md (hunk 3 above)
Comment (xyxxxxx) [reply README.md:12]: 下面的 `## 使用方法` 有提到“创建一个名为 mlperf、大小 2 Ti 的 PVC”，这里的 `## 使用前提` 是一个关于集群的大致描述

File: image-segmentation/workflow_DGXA100_40GB_4.yaml [hunk 8]
```diff
@@ -0,0 +1,204 @@
+apiVersion: batch.tensorstack.dev/v1beta1
```
Comment (xyxxxxx) [inline image-segmentation/workflow_DGXA100_40GB_4.yaml:1]: 受限于目前只有一种测试环境，只能提供 `workflow_DGXA100_40GB_4.yaml` 这一种配置。按道理来说应该进一步提供更多配置，比如 `workflow_DGXA100_80GB_4.yaml`、`workflow_DGXA100_80GB_16.yaml` 等等，以适应用户可能的不同硬件配置。

Comment (ppkube) [reply image-segmentation/workflow_DGXA100_40GB_4.yaml:1]: not a concern, as users can easily create their own

File: reinforcement-learning/mpijob.yaml (hunk 5 above)
Comment (xyxxxxx) [reply reinforcement-learning/mpijob.yaml:76]: https://github.com/mlcommons/training_results_v2.1/blob/main/NVIDIA/benchmarks/unet3d/implementations/mxnet-22.04/main.py#L43

训练脚本会往 `/results` 这个路径写日志，路径是写死的，必须挂一个卷在这个路径下，否则运行脚本会出错。与此同时我又不需要这一个日志，所以挂了个 emptyDir。

这算是一个 workaround。另一种方法是改一下脚本，加上这个路径。

Comment (ppkube) [reply reinforcement-learning/mpijob.yaml:76]: Got it. Let's keep the original training script as is.

File: README.md (hunk 1 above)
Comment (xyxxxxx) [reply README.md]: modified

File: image-segmentation/README.md (hunk 2 above)
Comment (xyxxxxx) [reply image-segmentation/README.md:25]: added

File: image-segmentation/workflow_DGXA100_40GB_4.yaml (hunk 4 above)
Comment (xyxxxxx) [reply image-segmentation/workflow_DGXA100_40GB_4.yaml]: fixed to `$(params.pvc_name)`

File: reinforcement-learning/mpijob.yaml (hunk 6 above)
Comment (xyxxxxx) [reply reinforcement-learning/mpijob.yaml]: modified

File: image-segmentation/workflow_DGXA100_40GB_4.yaml (hunk 7 above)
Comment (xyxxxxx) [reply image-segmentation/workflow_DGXA100_40GB_4.yaml]: modified

File: README.md [hunk 9]
```diff
@@ -1 +1,34 @@
...
+
+1. （可选）创建一个队列或选用一个现有队列，其应具有**充足的资源**和**适当的设置**。
+
+    这里**充足的资源**指队列有足以运行相应 benchmark 的 CPU 额度、内存额度、扩展资源 `nvidia.com/gpu` 等资源配额。不同的 benchmark 以及不同的配置需要不同的资源配额，请参阅各个 benchmark 的 README 文档。
+
+    **适当的设置**指队列的优先级、是否可被抢占以及节点筛选的设置。节点筛选一般应添加适当的标签匹配规则，以限制队列可以使用的节点，例如可以匹配 `sched.tensorstack.dev/accelerator: nvidia-gpu`、`sched.tensorstack.dev/node-alloc-mode: nvidia`、`nvidia.com/gpu.product: NVIDIA-A100-SXM-80GB`、`nvidia.com/gpu.memory: 81920` 等标签。
+
+    这一步是可选的，是否执行这一步取决于您是否需要对作业调度进行控制。例如，如果您不希望 Job 的工作器被调度到某一些节点上，那么这一步就是必要的。
+
+1. 在您的项目中创建一个名为 mlperf、大小 2 Ti 的 PVC，然后创建一个同样名为 mlperf 的 Notebook 挂载该 PVC，镜像和资源不限（如要使用远程操作，请开启 SSH）。
```
Comment (ppkube) [inline README.md]:
```suggestion
1. 在您的项目中创建一个名为 mlperf、大小 2 TiB 的 PVC，然后创建一个同样名为 mlperf 的 Notebook 挂载该 PVC，镜像和资源不限（如要使用远程操作，请开启 SSH）。
```

**Review by ppkube — APPROVED** [review]
LGTM

**Pull request merged**